client/src/PlayerLanding/PlayerInGame.tsx

```diff
@@ -62,13 +62,15 @@ export function PlayerInGame() {
         ),
     };
 
-    return nightOrder;
-  }, [script, game.playersToRoles]);
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const { Traveler, ...distributionsByPlayerCount } = {
```

Comment:
Why is this reusing the same useMemo???

Reply:
It doesn't even need to be memoized

Reply:
Eh it felt like a waste to repeat
```
const travelerCharacters = Object.values(game.playersToRoles)
    .map((role) => getCharacter(role))
    .filter((character) => character.team === "Traveler");
``` again outside the memo we've already got, but so be it.

Reply:
Don't preoptimize, it's stupid and causes bugs